plugins/ltac2/tac2intern.mli

```diff
@@ -19,6 +19,15 @@ val intern_typedef : (KerName.t * int) Id.Map.t -> raw_quant_typedef -> glb_quan
 val intern_open_type : raw_typexpr -> type_scheme
 val intern_notation_data : Id.Set.t -> raw_tacexpr -> Tac2env.notation_data
 
+val genintern_warn_not_unit : Genintern.glob_sign -> (Name.t * Tac2typing_env.mix_type_scheme) list ->
```

Comment:
This is ugly, can't you pass an optional argument or some flag in the function below?

Reply:
It should be ugly, warn_not_unit should be avoided IMO

Reply:
Meh, I'd still prefer having an optional flag that is recommended against.